app/js/components/labOrderEntry/LabDraftOrder.jsx

```diff
@@ -50,7 +50,7 @@ export class LabDraftOrder extends PureComponent {
               <a
                 id="draft-toggle-btn" href="#" onClick={event => this.handleToggleDraftOrderUgency(event, order)}
               >
-                <i className={iconClass} title="Urgency" />
+                <i className={iconClass}>&#x25B2;</i>
```

Comment:
What does this control code stand for?

Reply:
@mogoodrich it is for the urgency Icon, font-awesome did not have the Icon Dave wanted, so he suggested I use this, This is what it looks like.
![screen shot 2018-08-27 at 5 13 34 pm](https://user-images.githubusercontent.com/26272732/44671155-9a570300-aa1c-11e8-9cb7-3f905d457cd9.png)
![screen shot 2018-08-27 at 5 16 40 pm](https://user-images.githubusercontent.com/26272732/44671298-10f40080-aa1d-11e8-8afa-42d245c52d1d.png)